### PR TITLE
edited blog post turn R env into kernel and added image, also edited …

### PR DESCRIPTION
**Pull request by alwaysawn**
…blog post installing software

### CODE REVIEW

**Review by natalya-patrikeeva — CHANGES_REQUESTED** [review]
a few minor changes

File: docs/blog/posts/2022-11-14_installing_software.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    - Software
+authors:
+    - nrapstin
+---
+# Installing Software on the Yens
+
+As a yen user, you can install your own custom software in your home directory or any location where you have permissions
+(such as a shared project space). If you are working with other researchers on a shared project, it is a good idea to have 
+a dedicated shared `software` directory where you can install required software. 
+
```
[the remark this answers is not in the document]
Comment (alwaysawn) [reply docs/blog/posts/2022-11-14_installing_software.md:15]: done

File: docs/blog/posts/2022-11-14_installing_software.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+As an example, we can install a newer [Tesseract software](https://github.com/tesseract-ocr/tesseract){target="_blank"} which is free and is developed by Google.
```
[the remark this answers is not in the document]
Comment (alwaysawn) [reply docs/blog/posts/2022-11-14_installing_software.md:16]: done

File: docs/blog/posts/2022-11-14_installing_software.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+Then we download the source code to the yens with `wget` command and pasting the link to the source code.
+For this example, I am using a shared directory in `/zfs/gsb/intermediate-yens/software` but choose a shared location
+where you want the software to be installed and use the correct path to it instead.
+
+First, make directory where you want the binary to be (this should be a shared project space if you want
+other research to be able to use the software):
+ 
+```title="Terminal Command"
+mkdir /zfs/gsb/intermediate-yens/software/tesseract-5.2.0
```
[the remark this answers is not in the document]
Comment (alwaysawn) [reply docs/blog/posts/2022-11-14_installing_software.md:35]: done

[the remark this answers is not in the document]
Comment (alwaysawn) [reply docs/blog/posts/2022-11-14_installing_software.md:35]: done

File: docs/blog/posts/2022-11-14_installing_software.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+```
+
+Navigate where you want to download the source file to (you can also download the source to your home):
+
+```title="Terminal Command"
+cd /zfs/gsb/intermediate-yens/software
```
[the remark this answers is not in the document]
Comment (alwaysawn) [reply docs/blog/posts/2022-11-14_installing_software.md:41]: done

File: docs/blog/posts/2023-08-23-howRenvJupyterKernel.md [hunk 5]
```diff
@@ -32,15 +41,15 @@ Platform: x86_64-pc-linux-gnu (64-bit)
...
 Exit out of your instance of `R` and check for the existence of your new Jupyter kernel.
 
-```bash
-$ ml anaconda3
-$ jupyter kernelspec list
-
+```title="Terminal Command"
+jupyter kernelspec list
+```
+```{ .yaml .no-copy }
```
[the remark this answers is not in the document]
Comment (alwaysawn) [reply docs/blog/posts/2023-08-23-howRenvJupyterKernel.md:47]: done

File: docs/blog/posts/2022-11-14_installing_software.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+with `--prefix` argument in `configure` execution (use the directory path you made above):
+
+```title="Terminal Command"
+./autogen.sh
+./configure --prefix=/zfs/gsb/intermediate-yens/software/tesseract-5.2.0
+make
+make install
+```
+
+If everything completes successfully, add a path to the bin directory in your bash profile.
```
[the remark this answers is not in the document]
Comment (alwaysawn) [reply docs/blog/posts/2022-11-14_installing_software.md:65]: done

File: docs/blog/posts/2022-11-14_installing_software.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+Tesseract also needs to have an English tessdata file (or other language train data) to use language data models as described [here](https://github.com/tesseract-ocr/tessdata){target="_blank"}. We download [`eng.traineddata`](https://github.com/tesseract-ocr/tessdata/blob/main/eng.traineddata){target="_blank"} file and copy it to `tesseract-5.2.0/tessdata` directory.
+
+Now, we can call `tesseract` executable from anywhere on the yen's file system.
+
+Source the bash profile to execute the added `export PATH` command:
+```title="Terminal Command"
+source ~/.bash_profile
+```
+
+Check the new tesseract version:
```
[the remark this answers is not in the document]
Comment (alwaysawn) [reply docs/blog/posts/2022-11-14_installing_software.md:79]: done

File: docs/blog/posts/2023-08-23-howRenvJupyterKernel.md [hunk 8]
```diff
@@ -1,25 +1,34 @@
...
 date:
   created: 2023-08-23
+categories:
+    - R
+    - Environments
+authors:
+    - mason
 ---
 
 # How do I export my R environment to JupyterHub?
```
Comment (natalya-patrikeeva) [inline docs/blog/posts/2023-08-23-howRenvJupyterKernel.md:11]: For consistency, we should capitalize every word in the titles

File: docs/blog/posts/2022-11-14_installing_software.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+  created: 2022-11-14
+categories:
+    - Yen
+    - Software
+authors:
+    - nrapstin
+---
+# Installing Software on the Yens
+
+As a yen user, you can install your own custom software in your home directory or any location where you have permissions
```
Comment (natalya-patrikeeva) [inline docs/blog/posts/2022-11-14_installing_software.md:12]: I think we agreed on convention to use `yen` every time (not Yen, yen, or other variations)

File: docs/blog/posts/2023-08-23-howRenvJupyterKernel.md [hunk 10]
```diff
@@ -1,25 +1,34 @@
...
-$ /path/to/virtual/environment/bin/R
+```title="Terminal Command"
+/path/to/virtual/environment/bin/R
 ```
 
 ### Install and Use `IRkernel`
 Once in your instance of `R`, install the `IRkernel` package and create a Jupyter kernel for your `R` environment.
 
-```R
+```{ .yaml .no-copy title="Terminal Output"}
```
Comment (natalya-patrikeeva) [inline docs/blog/posts/2023-08-23-howRenvJupyterKernel.md:31]: you can add .r to syntax highlight the output with R language

File: docs/blog/posts/2023-08-23-howRenvJupyterKernel.md [hunk 11]
```diff
@@ -32,15 +41,15 @@ Platform: x86_64-pc-linux-gnu (64-bit)
...
+```title="Terminal Command"
+jupyter kernelspec list
+```
+```{ .yaml .no-copy title="Terminal Output" }
 Available kernels:
 rtest                 /home/users/mpjiang/.local/share/jupyter/kernels/rtest
 ```
 
 ### Use Kernel in JupyterHub
 Open a JupyterHub session in your browser and you should now find your R environment available as a kernel under the `displayname` that you chose:
```
Comment (natalya-patrikeeva) [inline docs/blog/posts/2023-08-23-howRenvJupyterKernel.md:53]: link to Jupyter Page here?

**Review by natalya-patrikeeva — CHANGES_REQUESTED** [review]
last changes, should be minor!

File: docs/blog/posts/2023-08-23-howRenvJupyterKernel.md (hunk 8 above)
Comment (alwaysawn) [reply docs/blog/posts/2023-08-23-howRenvJupyterKernel.md:11]: done

File: docs/blog/posts/2023-08-23-howRenvJupyterKernel.md (hunk 10 above)
Comment (alwaysawn) [reply docs/blog/posts/2023-08-23-howRenvJupyterKernel.md:31]: done

File: docs/blog/posts/2023-08-23-howRenvJupyterKernel.md (hunk 11 above)
Comment (alwaysawn) [reply docs/blog/posts/2023-08-23-howRenvJupyterKernel.md:53]: done

File: docs/blog/posts/2022-11-14_installing_software.md (hunk 9 above)
Comment (alwaysawn) [reply docs/blog/posts/2022-11-14_installing_software.md:12]: done

**Pull request merged**